feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/data/GovernanceStateUpdater.kt

```diff
@@ -0,0 +1,7 @@
+package io.novafoundation.nova.feature_governance_api.data
+
+import io.novafoundation.nova.runtime.multiNetwork.chain.model.Chain
+
+interface GovernanceStateUpdater {
```

Comment:
Need to provide this to Root Module

Reply:
I suggest to name it somehow different because `SomethingSomethingUpdater` creates association with UpdateSystem, at least for me\
Maybe something like `MutableGovernanceState`?